src/views/SignUpView.vue

```diff
@@ -47,16 +90,105 @@
 </template>
 
 <script>
+import { mapActions } from 'pinia'
+import { useUserStore } from '../stores/user'
 import TheForm from '../components/shared/TheForm.vue'
 import TheInput from '../components/shared/TheInput.vue'
 import TheButton from '../components/shared/TheButton.vue'
+import ErrorToast from '../components/shared/ErrorToast.vue'
 
 export default {
   name: 'SignUpView',
   components: {
     TheForm,
     TheInput,
-    TheButton
+    TheButton,
+    ErrorToast
+  },
+  data() {
+    return {
+      userData: {
+        name: '',
+        passOne: '',
+        passTwo: '',
+        email: ''
+      },
+      errorMessages: {
+        nameError: '',
+        passOneError: '',
+        passTwoError: '',
+        emailError: '',
+        checkboxError: ''
+      },
+      isCheckboxChecked: false,
+      isFormSubmitted: false,
+      isToastShown: false
+    }
+  },
+  methods: {
+    ...mapActions(useUserStore, ['singUp']),
+    // eslint-disable-next-line consistent-return
+    checkLength(input, minChar, label) {
+      if (input === '') {
+        return `The "${label}" value is required.`
+      }
+
+      if (input !== '' && input.length < minChar) {
+        return `${label} must have at least ${minChar} characters.`
+      }
+
+      return ''
+    },
```

Comment:
I think you can remove that eslint-disable line as return is consistent now

Reply:
🙈
Didn't notice, thank you.

**Corrected**